wolfcrypt/src/pkcs7.c

```diff
@@ -6051,7 +6069,8 @@ int wc_PKCS7_SetSignerIdentifierType(PKCS7* pkcs7, int type)
         return BAD_FUNC_ARG;
 
     if (type != CMS_ISSUER_AND_SERIAL_NUMBER &&
-        type != CMS_SKID) {
+        type != CMS_SKID &&
+        type != DEGENERATE_SID) {
```

Comment:
We should update the comment on wc_PKCS7_SetSignerIdentifierType() to mention why/when type would be set to DEGENERATE_SID.

Reply:
Thanks for the review, am adding a commit to update the comments.